.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
```

Comment:
We need to test on OSX too, since our two supported platforms are Windows and OSX.

Reply:
Added